search.js

```diff
@@ -3,7 +3,20 @@
 app({
   appId: 'latency',
   apiKey: '6be0576ff61c053d5f9a3225e2a90f76',
-  indexName: 'instant_search'
+  indexName: 'instant_search',
+  searchParameters: {
+    hitsPerPage: 10,
+  },
+  searchFunction(helper) {
+    if (helper.state.query.length > 0) {
+      document.querySelector('.ais-search-box--magnifier').style.display =
```

Comment:
I'm not sure I understand the motivation of this implementation?

Reply:
On https://preview.algolia.com/instantsearch/, the searchbox looks like this:

![image](https://user-images.githubusercontent.com/123822/33668220-69b5ff78-da9f-11e7-83ab-e9585948db63.png)

The magnifying glass is then replaced with the cross when the query is not empty.